programs/mmm/src/util.rs

```diff
@@ -783,6 +785,68 @@ pub fn check_allowlists_for_mint_ext(
     Err(MMMErrorCode::InvalidAllowLists.into())
 }
 
+pub fn check_allowlists_for_mpl_core<'info>(
+    allowlists: &[Allowlist],
+    asset: &Box<Account<'info, IndexableAsset>>,
```

Comment:
nit: should be able to just 
```suggestion
    asset: &IndexableAsset,
```

Reply:
interesting,